koku/masu/processor/parquet/ocp_cloud_parquet_report_processor.py

```diff
@@ -234,14 +242,21 @@ def process(self, parquet_base_filename, daily_data_frames):
                 cluster_topology = accessor.get_openshift_topology_for_multiple_providers(ocp_provider_uuids)
             # Get matching tags
             matched_tags = self.get_matched_tags(ocp_provider_uuids)
-            for i, daily_data_frame in enumerate(daily_data_frames):
-                openshift_filtered_data_frame = self.ocp_on_cloud_data_processor(
-                    daily_data_frame, cluster_topology, matched_tags
-                )
+            daily_data_frames = pd.concat(daily_data_frames, ignore_index=True)
```

Comment:
Our current processing flow, during convert to parquet we store daily_data_frames in a list all in memory then previously iterated over that list. The issue there was the df's in this list of df's likely contained dates for any of the range we processed, meaning our batching {i} count would not be consistent between runs.

Since we are storing all these df's in memory anyway we might as well concat them together and process them all at the same time. That way inside `create_partitioned_ocp_on_cloud_parquet` when we partition the df on usage dates we actually get all the dates for the current open df's in a single pass. Simplifying the batch numbers here and between multiple workers.

Reply:
So we are overwriting the original `daily_data_frames` var so memory _shouldnt_ change at all. I did a quick test locally and it didnt seem to make my pod memory go up at all.